drivers/network/afd/afd/write.c

```diff
@@ -334,6 +336,8 @@ AfdConnectedSocketWriteData(PDEVICE_OBJECT DeviceObject, PIRP Irp,
     PAFD_SEND_INFO SendReq;
     UINT TotalBytesCopied = 0, i, SpaceAvail = 0, BytesCopied, SendLength;
     KPROCESSOR_MODE LockMode;
+    INT FullSendLen, LoopIdx; // accumulator for full packet length
+    char pktbuf[PAD_BUFFER];  // local packet assembly buffer   
```

Comment:
```suggestion
    char pktbuf[PAD_BUFFER];  // local packet assembly buffer
```
I do recommend to use the Notepad++ function again, so there is no chance to miss any EOL whitespace.
I will not point to further hits, I used this suggestion just as an example to highlight at least one hit.

Reply:
ran it through notepad plus and cleaned that as well as other training spaces. also rechecked to make sure there are no tab indents.

Reply:
I do confirm that you fixed that fine. Resolved.